src/Monolog/Processor/ProcessorInterface.php

```diff
@@ -19,7 +19,7 @@
 interface ProcessorInterface
 {
     /**
-     * @return array The processed records
+     * @return array The processed record
      */
-    public function __invoke(array $records);
+    public function __invoke(array $record);
```

Comment:
Ideally I would also add a return type, but that could be considered BC break. Let me know if you want me to add it.
```php
public function __invoke(array $record): array;
```